packages/expo-location/ios/EXLocation/TaskConsumers/EXLocationTaskConsumer.m

```diff
@@ -0,0 +1,85 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <CoreLocation/CLLocationManager.h>
+
+#import <EXLocation/EXLocation.h>
+#import <EXLocation/EXLocationTaskConsumer.h>
+#import <EXTaskManagerInterface/EXTaskInterface.h>
+
+@interface EXLocationTaskConsumer ()
+
+@property (nonatomic, strong) CLLocationManager *locationManager;
+@property (nonatomic, strong) NSDictionary *options;
+
+@end
+
+@implementation EXLocationTaskConsumer
+
+- (void)dealloc
+{
+  [self reset];
+}
+
+# pragma mark - EXTaskConsumerInterface
+
+- (void)didRegisterTask:(id<EXTaskInterface>)task
+{
+  _task = task;
+  _locationManager = [CLLocationManager new];
+
+  _locationManager.delegate = self;
+  _locationManager.allowsBackgroundLocationUpdates = YES;
+  _locationManager.pausesLocationUpdatesAutomatically = NO;
```

Comment:
I think we might want to eventually have this configurable, since allowing pausing can improve battery life. But this is probably fine for a first pass

Reply:
Yeah, I agree it might be useful, but maybe we can do it in the future as it would also require adding `activityType` option. I don't really want to make this PR constantly growing...